<launch>

	<!-- Machine -->
	<machine name="localhost" address="localhost" env-loader="/opt/ros/hydro/env.sh"/>
	<arg name="machine" default="localhost" />
	
	<!-- load obstacle map configurations from YAML file to parameter server -->
	<rosparam file="$(find dwl_terrain)/config/obstacle_map.yaml" command="load"/>
	
	<node pkg="dwl_terrain" type="obstacle_map_server" name="terrain_obstacle_map" output="screen" machine="$(arg machine)">
		<remap from="obstacle_map" to="/obstacle_map" />
		<param name="world_frame" type="string" value="odom" />
		<!-- Base frame of the robot -->
		<param name="base_frame" type="string" value="base_link" />
	</node>
	
</launch>
